app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,20 @@
+class CommentsController < ApplicationController
+    def create
+        @post = Post.find(params[:post_id])
+        @comment = @post.comments.new(comment_params)
+        @comment.author = current_user
+
+        if @comment.save
+            flash[:notice] = 'Comment is Ceated!'
+        else
+            flash[:alert] = 'Comment is not Created!'
+        end
+        redirect_to @post
+    end
+
+    private
+
+    def comment_params
+        params.require(:comment).permit(:text)
+    end
+end
```

Comment:
As required by the previous CR, we need to create a NEW method to be able to create the view form to add comments. As you've commented it will be necessary a new page for that reason you will need to update your routes to have something like this 👇🏽 

```ruby
POST   /posts/:post_id/comments(.:format)     comments#create
 GET   /posts/:post_id/comments/new(.:format)     comments#new
```

- [ ] Please 🙏🏽 create the corresponding view for adding a new comment as requested by the previous CR it is necessary to be able to create a new comment.